src/Core/src/Platform/Windows/ContentPanel.cs

```diff
@@ -96,18 +96,21 @@ public void UpdateBackground(Paint? background)
 			_borderPath.UpdateBackground(background);
 		}
 
-		public void UpdateBorderShape(IShape borderShape)
+		public void UpdateBorderStroke(IBorderStroke borderStroke)
```

Comment:
This is a breaking change (changing the name of the public method). Maybe `UpdateBorderStroke` should be `internal`, and `UpdateBorderShape` should remain, but do nothing and be marked obsolete.

Reply:
Updated to avoid the breaking change.